packages/dataviews/src/utils.ts

```diff
@@ -20,17 +20,6 @@ export function sanitizeOperators< Item extends AnyItem >(
 		operators = [ OPERATOR_IS_ANY, OPERATOR_IS_NONE ];
 	}
 
-	// Transform legacy in, notIn operators to is, isNot.
-	// To be removed in the future.
-	if ( operators.includes( 'in' ) ) {
-		operators = operators.filter( ( operator ) => operator !== 'is' );
-		operators.push( 'is' );
-	}
-	if ( operators.includes( 'notIn' ) ) {
-		operators = operators.filter( ( operator ) => operator !== 'notIn' );
-		operators.push( 'isNot' );
-	}
```

Comment:
I had confirmed previously with @oandregal that these are not used anymore.

Reply:
I've prepared a PR that adds a changelog note about this, to let consumers of the package know about the breaking changes:  https://github.com/WordPress/gutenberg/pull/62013